daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/EvaluationOrderTest.scala

```diff
@@ -1869,7 +1869,7 @@ class EvaluationOrderTest extends AnyFreeSpec with Matchers with Inside {
             val (res, msgs) = evalUpdateApp(
               pkgs,
               e"""\(exercisingParty : Party) ->
-             ubind cId: ContractId M:Human <- create @M:Human M:Human {person = exercisingParty, ob = exercisingParty, ctrl = exercisingParty, precond = True, key = M:toKey exercisingParty, nested = M:buildNested 0} in
+             ubind cId: ContractId M:Human <- create @M:Human M:Human {person = exercisingParty, obs = exercisingParty, ctrl = exercisingParty, precond = True, key = M:toKey exercisingParty, nested = M:buildNested 0} in
```

Comment:
wait why did these change?

Reply:
Because the scala test code misspelled the field names as `ob`. The daml code spells them `obs`.

Reply:
Gotcha